packages/apps/ability-uniswap-swap/project.json

```diff
@@ -29,6 +29,15 @@
         "cwd": "{projectRoot}"
       }
     },
+    "action:create-pkp": {
+      "executor": "nx:run-commands",
+      "dependsOn": ["action:deploy"],
+      "options": {
+        "commands": ["node scripts/create-pkp-for-prepare.js", "nx run ability-uniswap-swap:build"],
+        "cwd": "{projectRoot}",
+        "parallel": false
+      }
+    },
```

Comment:
This new NX target must be manually ran anytime anytime `packages/apps/ability-uniswap-swap/src/lib/prepare/lit-action.ts` is changed to produce a new signer PKP